admin/acceptance_tests/user_test.py

```diff
@@ -150,3 +166,39 @@ def test_selenium(self, dbsession, selenium):
         dbsession.expire(user)
         assert user.username == 'new_name_éôù'
         assert user.email == 'new_email'
+
+    # in order to make this work, had to install selenium gecko driver
+    @pytest.mark.usefixtures("selenium", "selenium_app")
+    def test_delete_selenium(self, dbsession, selenium):
+        from c2cgeoportal_commons.models.main import User
+        user_id = dbsession.query(User). \
+            filter(User.username == 'babar_13'). \
+            one().id
+        selenium.get('http://127.0.0.1:6543' + '/user/')
+        elem = selenium.find_element_by_xpath("//div[@class='infos']")
+        assert 'Showing 1 to 10 of 23 entries' == elem.text
+        main_window_handle = selenium.current_window_handle
+        elem = selenium.find_element_by_xpath("//button[@title='Refresh']/following-sibling::*")
+        elem.click()
+        elem = selenium.find_element_by_xpath("//a[contains(.,'50')]")
+        elem.click()
+        elem = selenium.find_element_by_xpath("//a[contains(@href,'" +
+                                              str(user_id) +
+                                              "/edit')]/following-sibling::*")
+        elem.click()
+        selenium.switch_to_alert().accept()
+
+        selenium.switch_to_default_content()
+        selenium.refresh()
+        elem = selenium.find_element_by_xpath("//div[@class='infos']")
+        assert 'Showing 1 to 10 of 22 entries' == elem.text
+        elem = selenium.find_element_by_xpath("//button[@title='Refresh']/following-sibling::*")
+        elem.click()
```

Comment:
Is it stricly neccessary to do such many refreshes

Reply:
Seems that gecko driver need a little time to shift focus exiting alert window. I Prefer calling refresh than adding tempo.